src/steps/opportunity/opportunity-field-equals.ts

```diff
@@ -82,6 +122,24 @@ export class OpportunityFieldEquals extends BaseStep implements StepInterface {
       return this.error('There was an error during validation of opportunity field: %s', [e.message]);
     }
   }
+
+  createRecord(opportunity: Record<string, any>) {
+    Object.keys(opportunity.attributes).forEach(attr => opportunity[titleCase(attr)] = opportunity.attributes[attr]);
+    delete opportunity.attributes;
+    return this.keyValue('opportunity', 'Checked Opportunity', opportunity);
```

Comment:
- [ ] Strangely, I am still seeing `attributes` come through.

![Screen Shot 2020-02-25 at 1 52 50 PM](https://user-images.githubusercontent.com/3496491/75290942-21ce2f80-57d6-11ea-9cad-95dc9a5775a2.png)

Reply:
The `createRecord` was not invoked :(